koji_wrapper/base.py

```diff
@@ -16,6 +16,13 @@ def __init__(self, url='', topurl='', session=None):
         self.topurl = topurl
         self.session = session
 
+        if isinstance(session, KojiWrapperBase):
```

Comment:
I think this should just go in the session setter, we already have logic here, just not the setting of url/topurl.  Otherwise, seems reasonable